spec/models/video_spec.rb

```diff
@@ -2,6 +2,60 @@
 
 describe Video do
   it { should belong_to(:category) }
-  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:title) }
   it { should validate_presence_of(:description) }
+
+  describe "search_by_title" do 
+    it 'if none of the videos found, return an empty array' do 
```

Comment:
You could use `context` here instead of `if none of the videos found` in the message. Check out [here](http://betterspecs.org/#contexts).